MANIFEST.in

```diff
@@ -0,0 +1 @@
+include gmso/lib/jsons/*.json
```

Comment:
Surely there  is more we need in this file? Unrelated to these changes

Reply:
I think so. Should be added in another PR.

Reply:
I suggest adding the license file, if nothing else. This ensures that distributions of your package include the license, which is required by conda-forge.